hack/local-up-cluster.sh

```diff
@@ -386,17 +386,21 @@ kind: Namespace
 metadata:
   name: kube-system
 EOF
-        ${KUBECTL} config set-cluster local --server=http://127.0.0.1:8080 --insecure-skip-tls-verify=true
+        ${KUBECTL} config set-cluster local --server=http://${API_HOST}:${API_PORT} --insecure-skip-tls-verify=true
         ${KUBECTL} config set-context local --cluster=local
         ${KUBECTL} config use-context local
 
         ${KUBECTL} create -f namespace.yaml
         # use kubectl to create skydns rc and service
-        ${KUBECTL} --namespace=kube-system create -f skydns-rc.yaml 
-        ${KUBECTL} --namespace=kube-system create -f skydns-svc.yaml
-        echo "Kube-dns rc and service successfully deployed."
+        echo "Starting kube DNS RC and service..."
+        ${KUBECTL} --namespace=kube-system create -f skydns-rc.yaml && \
+            ${KUBECTL} --namespace=kube-system create -f skydns-svc.yaml
+        if [ $? -eq 0 ]; then
```

Comment:
Maybe just execute the command in the conditional so it'll do the same thing no matter whether -e is set or not? It doesn't matter to me too much as long as still works :)